lib/PuppeteerSharp.Tests/CSSCoverageTests/CSSCoverageTests.cs

```diff
@@ -63,6 +63,11 @@ public async Task ShouldReportStylesheetsThatHaveNoCoverage()
         {
             await Page.Coverage.StartCSSCoverageAsync();
             await Page.GoToAsync(TestConstants.ServerUrl + "/csscoverage/unused.html");
+
```

Comment:
This was the test that was failing some of the time. I'm not sure if I changed something to cause it to fail, or if this was failing before due to a race.

Reply:
We shouldn't change this test.

Reply:
Reverted. I retested locally against both this PR and against master, and after 10 tried couldn't get it to fail, so I'm not sure why it was failing before on this PR.